app/views/book_tours/create.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if @book_tour.errors.any? %>
+   alert('<%= t("booking_fail") %>')
```

Comment:
Cần alert ra cụ thể message để họ biết lỗi do đâu chứ k message chung chung thế này.
Có thể là lấy `error.full_messages.first` cũng đc

Sau này có thể implement in ra list lỗi luôn. Đặt 1 cái ull li ở trên cái div của form rồi append ra. Kieeur đó